addons/website/static/tests/tours/drop_404_ir_attachment_url.js

```diff
@@ -1,53 +1,74 @@
 /** @odoo-module **/
 
+import { waitFor } from "@odoo/hoot-dom";
 import {
     changeOption,
-    dragNDrop,
     registerWebsitePreviewTour,
-} from '@website/js/tours/tour_utils';
+} from "@website/js/tours/tour_utils";
 
-registerWebsitePreviewTour('drop_404_ir_attachment_url', {
-    test: true,
-    url: '/',
-    edition: true,
-}, () => [
-    ...dragNDrop({
-        id: 's_404_snippet',
-        name: '404 Snippet',
-        groupName: "Images",
-    }),
+registerWebsitePreviewTour(
+    "drop_404_ir_attachment_url",
     {
-        content: 'Click on the snippet image',
-        trigger: ':iframe .s_404_snippet img',
-        run: "click",
+        test: true,
+        url: "/",
+        edition: true,
     },
-    {
-        trigger: ".snippet-option-ReplaceMedia",
-    },
-    {
-        content: 'Once the image UI appears, check the image has no size (404)',
-        trigger: ':iframe .s_404_snippet img',
-        run() {
-            const imgEl = this.anchor;
-            if (!imgEl.complete
-                || imgEl.naturalWidth !== 0
-                || imgEl.naturalHeight !== 0) {
-                throw new Error('This is supposed to be a 404 image');
-            }
+    () => [
+        {
+            trigger: ".o_website_preview.editor_enable.editor_has_snippets",
+            noPrepend: true,
```

Comment:
Is not this something that is automatically done by `registerWebsitePreviewTour`?